### DISCUSSION TITLE
Keycloak Guide Maven issue

### DISCUSSION THREAD

**Discussion by ChandanPrasad12345** [keycloak/keycloak#25870] [Q&A]
Hi Team,
We are facing an issue while compiling the Keycloak Guide project.Below are the errors :
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  04:00 min
[INFO] Finished at: 2024-01-03T01:11:02-08:00
[INFO] ------------------------------------------------------------------------
[WARNING]
[WARNING] Plugin validation issues were detected in 6 plugin(s)
[WARNING]
[WARNING]  * org.apache.maven.plugins:maven-resources-plugin:3.2.0
[WARNING]  * org.apache.maven.plugins:maven-compiler-plugin:3.8.1
[WARNING]  * org.apache.maven.plugins:maven-clean-plugin:3.1.0
[WARNING]  * org.codehaus.mojo:buildnumber-maven-plugin:1.4
[WARNING]  * com.google.code.maven-replacer-plugin:maven-replacer-plugin:1.4.1
[WARNING]  * org.apache.maven.plugins:maven-enforcer-plugin:3.0.0-M3
[WARNING]
[WARNING] For more or less details, use 'maven.plugin.validation' property with one of the values (case insensitive): [BRIEF, DEFAULT, VERBOSE]
[WARNING]
[ERROR] Failed to parse plugin descriptor for org.keycloak:keycloak-guides-maven-plugin:999.0.0-SNAPSHOT (/dpshares/keycloak_upgrade_23_0_3/docs/maven-plugin/target/classes): No plugin descriptor found at META-INF/maven/plugin.xml -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginDescriptorParsingException

Can someone help here?

**Comment by consideRatio** [comment]
I ran into a missing `META-INF/maven/plugin.xml` file (but for theme-verifier) when doing `./mvnw compile -DskipTests`, but I got past it if I did `./mvnw clean install -DskipTests` - then the `META-INF/maven/plugin.xml` file etc got generated.
